samples/maps/demo/data-class-two-ranges/demo.css

```diff
@@ -10,3 +10,42 @@
     text-align: center;
     color: gray;
 }
+
+#popup-pie {
+    width: 290px;
+    height: 240px;
+    padding: 0.5em 1em;
+    background: #fff;
+    border: 1px solid #c5c5c5;
```

Comment:
I'm not sure, hasn't worked with CSS for some time, but I think we have colors variables defined in Highcharts. You should make a use of them

Reply:
this is also not needed here actually, removed it